servant-client/test/Servant/Client/TestServer.hs

```diff
@@ -0,0 +1,12 @@
+{-# LANGUAGE CPP                    #-}
+
+module Servant.Client.TestServer (
+  buildTestServer,
+  withTestServer,
+)where
+
```

Comment:
Hmm, now I added the comment to `Servant.ClientSpec.withTestServer`. @jkarni: Would you prefer it here?